src/components/GameCard.js

```diff
@@ -1,10 +1,15 @@
-import React from 'react';
+import React, { useContext } from 'react';
+import { GlobalContext } from '../context/GlobalContext';
 import '../css/GameCard.css';
 
-const GameCard = () => {
+const GameCard = ({ gameName, winner }) => {
+  const { games } = useContext(GlobalContext)
```

Comment:
You can name each context whatever you want. I wonder if `GameContext` is more descriptive than `GlobalContext` since all Context is Global?